src/commands/CmdInfo.cpp

```diff
@@ -116,9 +158,27 @@ int CmdInfo::execute(std::string& output) {
     auto description = task.get("description");
     auto indent = Context::getContext().config.getInteger("indent.annotation");
 
-    for (auto& anno : task.getAnnotations())
-      description += '\n' + std::string(indent, ' ') +
-                     Datetime(anno.first.substr(11)).toString(dateformatanno) + ' ' + anno.second;
+    {
+      const size_t termWidth = Context::getContext().getWidth();
+      const size_t approxNameCol = 20;  // conservative; covers labels like "Last modified"
+      const size_t valueWidth = termWidth > approxNameCol ? termWidth - approxNameCol : termWidth;
```

Comment:
Hm, this seems a bit of a hack, as it's "guessing" the size of the `[Table](https://github.com/GothenburgBitFactory/libshared/blob/master/src/Table.h)` column, and wrapping to that length. I suppose if the guess is incorrect you just get some weirdly line-broken text, so not a horrible bug. But maybe we could do better?

If that requires improvements to the libshared repo, that's fine -- it's only used by Taskwarrior and Timewarrior so we can modify it pretty easily.